scripts/get_helm.sh

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env bash
```

Comment:
I think we should do without the scripts folder, can you please remove it :) ?

Reply:
@commjoen should i unstage this folder?

Reply:
I didn't get you

Reply:
> @commjoen should i unstage this folder?

Yes please.